tests/foreman/ui/test_rhcloud_inventory.py

```diff
@@ -485,3 +485,79 @@ def test_rhcloud_global_parameters(
     hostnames = [host['fqdn'] for host in json_data['hosts']]
     assert virtual_host.hostname in hostnames
     assert baremetal_host.hostname in hostnames
+
+
+def test_subscription_connection_settings_ui_behavior(module_target_sat):
+    """Verify that the RH Cloud Inventory UI
+    reflects the subscription_connection_enabled setting
+
+    :id: 9b8648b5-0ffb-49c1-a19e-04a7a8ce896f
+
+    :steps:
+        1. Set the subscription_connection_enabled setting to true
+        2. Check that all the RH inventory settings, auto_upload and manual_upload descriptions,
+            cloud_connector and sync_status buttons are displayed in the UI
+        3. Set the setting to false
+        4. Set the subscription_connection_enabled setting to false
+        5. Verify that auto_update switch, auto_upload and manual_upload descriptions
+            and configure_cloud_connector and sync_all buttons are NOT displayed in the UI
+
+    :expectedresults:
+        1. The subscription_connection_enabled setting is reflected in the UI
+    """
+    with module_target_sat.ui_session() as session:
+        session.organization.select(org_name=DEFAULT_ORG)
+        session.location.select(loc_name=DEFAULT_LOC)
+
+        # Get the initial state of subscription_connection_enabled setting
+        initital_subs_conn_setting = module_target_sat.cli.Settings.list(
+            {'search': 'subscription_connection_enabled'}
+        )[0]['value']
```

Comment:
One extra nitpick for you - consider using `api` instead of `cli`, since it's just a setup thing for UI test. But it's rather matter of personal preference, non-blocking.